usaspending_api/spending_explorer/v2/filters/type_filter.py

```diff
@@ -28,22 +29,34 @@ def type_filter(_type, filters):
     # Get fiscal_date and fiscal_quarter
     for key, value in filters.items():
         if key == 'fy':
-            if value is not None:
-                fiscal_date, fiscal_quarter = generate_last_completed_fiscal_quarter(fiscal_year=value)
-            else:
+            try:
+                fiscal_year = int(value)
```

Comment:
Should this throw an error when a request fiscal year param is 'YY' such as `fy: '17'`?

Reply:
It never did before so I didn't want to add it now. If they want to search the year 17 I say we should let them. It'll give them no data

Reply:
If we're allowing it, this portion of the gist is inaccurate then (under Error Messages):
```
{
    "details": "Incorrect fiscal year format, should be four digit year: YYYY"
}
```

Reply:
I don't see that error actually in the code, but I do think we should enforce a 4 digit year.